cert_manager/certbot.py

```diff
@@ -25,7 +27,6 @@ def request_cert(self, domains):
             "--non-interactive",
             "--agree-tos",
             "--keep",
-            "--allow-subset-of-names",
```

Comment:
@ahmedbilal, this has to be dropped for the time being till the conversation about this is completed.